### PR TITLE
Update Country list in edit.js

### PR DESCRIPTION
**Pull request by QuinnWuest**

### CODE REVIEW

File: src/views/profile/edit.ejs [hunk 1]
```diff
@@ -114,19 +114,19 @@
...
         <option value="CK">Cook Islands</option>
         <option value="CR">Costa Rica</option>
         <option value="CI">Côte d'Ivoire</option>
         <option value="HR">Croatia</option>
         <option value="CU">Cuba</option>
         <option value="CW">Curaçao</option>
         <option value="CY">Cyprus</option>
-        <option value="CZ">Czech Republic</option>
+        <option value="CZ">Czechia</option>
+        <option value="CD">Democratic Republic of the Congo</option>
```
Comment (vck3000) [inline src/views/profile/edit.ejs:129]: Should this just be Congo? Having "Democratic" at the front would change the alphabetic ordering and might make it harder to find?

File: src/views/profile/edit.ejs [hunk 2]
```diff
@@ -114,19 +114,19 @@
...
-        <option value="CD">Congo, the Democratic Republic of the</option>
         <option value="CK">Cook Islands</option>
         <option value="CR">Costa Rica</option>
         <option value="CI">Côte d'Ivoire</option>
         <option value="HR">Croatia</option>
         <option value="CU">Cuba</option>
         <option value="CW">Curaçao</option>
         <option value="CY">Cyprus</option>
-        <option value="CZ">Czech Republic</option>
+        <option value="CZ">Czechia</option>
```
Comment (vck3000) [inline src/views/profile/edit.ejs:128]: Is Czech Republic more official? 
https://en.wikipedia.org/wiki/Czech_Republic

File: src/views/profile/edit.ejs [hunk 3]
```diff
@@ -164,14 +165,14 @@
         <option value="GY">Guyana</option>
         <option value="HT">Haiti</option>
         <option value="HM">Heard Island and McDonald Islands</option>
-        <option value="VA">Holy See (Vatican City State)</option>
+        <option value="VA">Holy See</option>
```
Comment (vck3000) [inline src/views/profile/edit.ejs:168]: Hmm, I kinda liked having the bracket here lol. What do you think?

File: src/views/profile/edit.ejs [hunk 4]
```diff
@@ -247,57 +248,58 @@
         <option value="PT">Portugal</option>
         <option value="PR">Puerto Rico</option>
         <option value="QA">Qatar</option>
+        <option value="CG">Republic of the Congo</option>
```
Comment (vck3000) [inline src/views/profile/edit.ejs:251]: Huh, looks like we have 3 Congos?

File: src/views/profile/edit.ejs [hunk 5]
```diff
@@ -93,14 +93,14 @@
         <option value="BJ">Benin</option>
         <option value="BM">Bermuda</option>
         <option value="BT">Bhutan</option>
-        <option value="BO">Bolivia, Plurinational State of</option>
+        <option value="BO">Bolivia</option>
         <option value="BQ">Bonaire, Sint Eustatius and Saba</option>
         <option value="BA">Bosnia and Herzegovina</option>
         <option value="BW">Botswana</option>
-        <option value="BV">Bouvet Island</option>
```
Comment (vck3000) [inline src/views/profile/edit.ejs:100]: This doesn't have an addition, did you mean to completely remove it?

File: src/views/profile/edit.ejs [hunk 6]
```diff
@@ -184,11 +185,10 @@
         <option value="KZ">Kazakhstan</option>
         <option value="KE">Kenya</option>
         <option value="KI">Kiribati</option>
-        <option value="KP">Korea, Democratic People's Republic of</option>
-        <option value="KR">Korea, Republic of</option>
+        <option value="XK">Kosovo</option>
```
Comment (vck3000) [inline src/views/profile/edit.ejs:188]: This looks new. Not sure if we have a flag icon for it?

**Review by vck3000 — COMMENTED** [review]
Looks like I did a bit of a sad thing and also had the list copied here:
https://github.com/vck3000/ProAvalon/blob/master/src/routes/profile.js#L403-L907

Could you update this list after you finish up the changes here? 

Maybe you could pull that list out into a separate ts file and have it be pulled in so that they share the same source of truth. If that's too much effort dw about it and just update the list to match haha. 

Thanks :).

File: src/views/profile/edit.ejs (hunk 5 above)
Comment (QuinnWuest) [reply src/views/profile/edit.ejs:100]: Oops

File: src/views/profile/edit.ejs (hunk 2 above)
Comment (QuinnWuest) [reply src/views/profile/edit.ejs:128]: Both are acceptable, but I believe ISO has it listed as Czechia

File: src/views/profile/edit.ejs (hunk 1 above)
Comment (QuinnWuest) [reply src/views/profile/edit.ejs:129]: The full name of this country is Democratic Republic of the Congo, not to be confused with (Republic of the) Congo

File: src/views/profile/edit.ejs (hunk 3 above)
Comment (QuinnWuest) [reply src/views/profile/edit.ejs:168]: I suppose the bracket could be readded

File: src/views/profile/edit.ejs (hunk 6 above)
Comment (QuinnWuest) [reply src/views/profile/edit.ejs:188]: Ah. There is a new one. I'll get a flag when i can.

File: src/views/profile/edit.ejs (hunk 4 above)
Comment (QuinnWuest) [reply src/views/profile/edit.ejs:251]: Should remove this one and keep Congo imo. DRC is a different country.

File: src/views/profile/edit.ejs (hunk 1 above)
Comment (vck3000) [reply src/views/profile/edit.ejs:129]: Ah interesting. Cool, let's keep it as is!

File: src/views/profile/edit.ejs (hunk 2 above)
Comment (vck3000) [reply src/views/profile/edit.ejs:128]: Hmm, could we keep it as Czech Republic then? Just anecdotally that's what I've been more familiar with. But if you think Czechia is more common/official let's go with that. I'll leave it up to you :).

File: src/views/profile/edit.ejs (hunk 4 above)
Comment (vck3000) [reply src/views/profile/edit.ejs:251]: Thanks!

File: src/views/profile/edit.ejs (hunk 6 above)
Comment (vck3000) [reply src/views/profile/edit.ejs:188]: Thanks!

**Pull request merged**